pkgs/applications/science/math/sage/default.nix

```diff
@@ -129,7 +129,7 @@ let
   singular = pkgs.singular.override { inherit flint; };
 
   maxima = pkgs.maxima.override {
-    lisp-compiler = pkgs.ecl.override {
+    lisp-compiler = pkgs.ecl.pkg.override {
```

Comment:
Why was the derivation moved to another attribute? That inconsistent with other ecosystems and triggers lots of noise changes.

Reply:
I will fix it